libraries/Wire/src/utility/twi.c

```diff
@@ -1026,7 +1026,7 @@ void HAL_I2C_ListenCpltCallback(I2C_HandleTypeDef *hi2c)
   /*  Previous master transaction now ended, so inform upper layer if needed
    *  then prepare for listening to next request */
   if ((obj->slaveMode == SLAVE_MODE_RECEIVE) && (obj->slaveRxNbData != 0)) {
-    obj->i2c_onSlaveReceive((uint8_t *) obj->i2cTxRxBuffer, obj->slaveRxNbData);
+    obj->i2c_onSlaveReceive(obj, (uint8_t *) obj->i2cTxRxBuffer, obj->slaveRxNbData);
```

Comment:
According to my first remarks:
```suggestion
    obj->i2c_onSlaveReceive(obj);
```

Reply:
Done